content/altinn-studio/guides/contributing/create-new-component/_index.en.md

```diff
@@ -0,0 +1,7 @@
+---
+title: Create a new component
+description: Create a new component
+weight: 10
+---
+
+## Create a new component in App-frontend
```

Comment:
Suggest we just add a `notice` shortcode with information that this information will come